packages/amazonq/src/lsp/config.ts

```diff
@@ -26,3 +46,7 @@ export function getAmazonQLspConfig(): ExtendedAmazonQLSPConfig {
         ...getServiceEnvVarConfig('amazonqLsp', Object.keys(defaultAmazonQLspConfig)),
     }
 }
+
+export function toAmazonQLSPLogLevel(logLevel: vscode.LogLevel): LspLogLevel {
```

Comment:
Lets explain why briefly: "Maps the VS Code log level to an equivalent Amazon Q language server log level, since they are not 1:1"